examples/templates/docker/main.tf

```diff
@@ -80,17 +80,10 @@ resource "docker_container" "workspace" {
   # Uses lower() to avoid Docker restriction on container names.
   name = "coder-${data.coder_workspace.me.owner}-${lower(data.coder_workspace.me.name)}"
   # Hostname makes the shell more user friendly: coder@my-workspace:~$
-  hostname = lower(data.coder_workspace.me.name)
-  dns      = ["1.1.1.1"]
+  hostname = data.coder_workspace.me.name
   # Use the docker gateway if the access URL is 127.0.0.1
-  command = [
-    "sh", "-c",
-    <<EOT
-    trap '[ $? -ne 0 ] && echo === Agent script exited with non-zero code. Sleeping infinitely to preserve logs... && sleep infinity' EXIT
-    ${replace(coder_agent.main.init_script, "/localhost|127\\.0\\.0\\.1/", "host.docker.internal")}
-    EOT
-  ]
-  env = ["CODER_AGENT_TOKEN=${coder_agent.main.token}"]
+  entrypoint = ["sh", "-c", replace(coder_agent.main.init_script, "/localhost|127\\.0\\.0\\.1/", "host.docker.internal")]
```

Comment:
I think we can replace these with the new 'provider.url' attribute.

Reply:
@ammario we can't. I never merged the change to adjust the hostname.

The primary problem was if you were using the tunnel, because the access URL would be `https://<url>:443`. If we just adjusted the hostname, then `host.docker.internal:443` would be dialed.

Reply:
Ah that would indeed have been neat. I guess we’ll keep it like this for now.